src/screens/Exchange/WBTCCafeInfo.js

```diff
@@ -0,0 +1,235 @@
+import Table, { TableRow, TableAmount } from 'components/Table';
+
+// utils, actions
+import { themedColors } from 'utils/themes';
+import { fontStyles, baseColors } from 'utils/variables';
+import { addWbtcPendingTxAction } from 'actions/exchangeActions';
+
+// models, constants
+import type { WBTCFeesWithRate, PendingWBTCTransaction } from 'models/WBTC';
+import { BTC, WBTC } from 'constants/assetsConstants';
+import { EXCHANGE_CONFIRM, WBTC_CAFE } from 'constants/navigationConstants';
+import type { Dispatch } from 'reducers/rootReducer';
+import { LIGHT_THEME } from 'constants/appSettingsConstants';
+
+// partials
+import WBTCSlippageModal from './WBTCSlippageModal';
+import WBTCCafeAddress from './WBTCCafeAddress';
+import { ExchangeIcon, TableWrapper } from './ConfirmationTable';
+import RenFeeIcon from './RenFeeIcon';
+
+type Props = {
+  wbtcData: ?WBTCFeesWithRate,
+  extendedInfo?: boolean,
+  navigation: NavigationScreenProp<*>,
+  amount?: string,
+  address?: string,
+  error?: boolean,
+  addWbtcPendingTx: (tx: PendingWBTCTransaction) => void,
+}
+
+const Row = styled.View`
+  flex-direction: row;
+  align-items: center;
+`;
+
+const Label = styled(BaseText)`
+  ${fontStyles.regular};
+  color: ${({ textColor }) => textColor || themedColors.secondaryText};
+  text-align: center;
+`;
+
+const ButtonWrapper = styled.View`
+  margin: 0px 0px 20px;
+`;
+
+const FeeRow = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 15px;
+`;
+
+const FeeWrapper = styled.View`
+  border-radius: 10px;
+  padding: 2px 10px;
+  margin-left: 10px;
+  background-color: ${({ theme }) =>
+    theme.current === LIGHT_THEME ? theme.colors.basic020 : theme.colors.primaryAccent220};
+`;
+
+const Fee = styled(BaseText)`
+  ${fontStyles.small};
+  color: ${baseColors.white};
+`;
+
+const Header = styled(MediumText)`
+  font-size: 18px;
+  margin-top: 30px;
+  margin-bottom: 16px;
+`;
+
+const WBTCCafeInfo = (props: Props) => {
+  const [maxSlippage, setMaxSlippage] = useState<number>(0.5);
+
+  const handleSlippagePress = () => Modal.open(() => (
+    <WBTCSlippageModal onModalWillHide={selectedVal => setMaxSlippage(selectedVal)} />
+  ));
+
+  const {
+    wbtcData, extendedInfo, navigation, amount, address, error, addWbtcPendingTx,
+  } = props;
+
+  const handleNextPress = () => {
+    if (!extendedInfo && !!navigation) {
+      return navigation.navigate(EXCHANGE_CONFIRM, {
+        wbtcTxData: { maxSlippage: maxSlippage / 100, amount: Number(amount) },
+        wbtcEstData: wbtcData,
+      });
+    }
+    Clipboard.setString(address || '');
+    return Toast.show({ message: t('toast.addressCopiedToClipboard'), emoji: 'ok_hand' });
+  };
+
+  const handleSent = () => {
+    addWbtcPendingTx({ amount: wbtcData?.estimate || 0, dateCreated: Date.now() });
+    navigation.navigate(WBTC_CAFE);
+  };
+
+  const getFeeNumber = () => {
+    if (!wbtcData) return 0;
```

Comment:
User would be left behind, maybe we can show at least some Toast, wdyt?

Reply:
Good catch. We'll still display 0, but we'll show a Toast and disable going any further (i.e. won't display BTC address)